projects/packages/publicize/src/jetpack-social-settings/class-settings.php

```diff
@@ -341,6 +341,7 @@ public function get_initial_state() {
 
 			$settings['is_publicize_enabled'] = true;
 			$settings['hasPaidFeatures']      = $publicize->has_paid_features();
+			$settings['hasJetpack']           = is_plugin_active( 'jetpack/jetpack.php' ) && $connection->has_connected_owner();
```

Comment:
I think you don't need this because that initial state is no longer used and it was replaced with the script-data that you used above.

Reply:
Done in https://github.com/Automattic/jetpack/pull/42000/commits/3399104a7afd85ce2f2bf521609af13a6175f574